contents.json

```diff
@@ -5485,6 +5485,12 @@
       "description": "A simple framework for easy iOS Toasts.",
       "homepage": "https://github.com/schmidyy/Loaf",
       "tags": ["toast", "alert"]
+    }, {
+      "title": "CircleBar",
+      "category": "menu",
```

Comment:
@perteraul it had better use the [tab](https://github.com/matteocrippa/awesome-swift#tab) category for this, will you?

Reply:
Just updated. it.